### PR TITLE
added logging quota overrides from the default, and added a specific quota targeting the openshift-logging namespace so that the customer may provision ES with PV's.

### PR DESCRIPTION
**Pull request by mwoodson**

### CODE REVIEW

File: deploy/osd-logging/03-storage-quota.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+- apiVersion: v1
+  kind: ResourceQuota
+  metadata:
+    name: logging-storage-quota
+    namespace: openshift-logging
+  spec:
+    hard:
+      requests.storage: "600G"
```
Comment (lisa) [inline deploy/osd-logging/03-storage-quota.yaml]:
```suggestion
      requests.storage: "600Gi"
```

I believe the unit is `Gi`, not `G`?

File: hack/00-osd-managed-cluster-config.selectorsyncset.yaml.tmpl [hunk 2]
```diff
@@ -1062,8 +1062,18 @@ objects:
...
           openshift.io/cluster-logging: 'true'
           openshift.io/cluster-monitoring: 'true'
+    - apiVersion: v1
+      kind: ResourceQuota
+      metadata:
+        name: logging-storage-quota
+        namespace: openshift-logging
+      spec:
+        hard:
+          requests.storage: "600G"
```
Comment (lisa) [inline hack/00-osd-managed-cluster-config.selectorsyncset.yaml.tmpl]:
```suggestion
          requests.storage: "600Gi"
```

Same as before about units.

File: deploy/osd-logging/01-namespace.yaml [hunk 3]
```diff
@@ -3,7 +3,9 @@ kind: Namespace
 metadata:
   name: openshift-logging
   annotations:
-    openshift.io/node-selector: "" 
+    openshift.io/node-selector: ""
   labels:
+    managed.openshift.io/service-lb-quota-exempt: "true"
```
Comment (lisa) [inline deploy/osd-logging/01-namespace.yaml]: Does the namespace need to be exempt for load balancers too?

**Comment by lisa** [comment]
/cancel hold

Removing my hold; though I still think we should re-use the framework for managing resource quotas that was added in #23, even though it will require extending it beyond its original intent.

**Comment by jewzaam** [comment]
/hold cancel
@lisa had wrong syntax

File: deploy/osd-logging/01-namespace.yaml (hunk 3 above)
Comment (jewzaam) [inline deploy/osd-logging/01-namespace.yaml]:
```suggestion
```

Comment (mwoodson) [reply deploy/osd-logging/01-namespace.yaml]: This is following the pattern of other namespaces having this exemption as well.  I followed that pattern here.

**Comment by lisa** [comment]
> /hold cancel
> @lisa had wrong syntax

Oops. Silly bot, why can't it do what I meant, and not what I told it to do? 😇

**Pull request merged**